src/sentry/options/defaults.py

```diff
@@ -573,6 +573,11 @@
 register("performance.issues.n_plus_one_api_calls.la-rollout", default=0.0)
 register("performance.issues.n_plus_one_api_calls.ea-rollout", default=0.0)
 register("performance.issues.n_plus_one_api_calls.ga-rollout", default=0.0)
+register("performance.issues.slow_span.problem-creation", default=0.0)
```

Comment:
I think this _might_ break! This PR both declares the option _and_ uses it. You're removing `SlowSpanDetector.is_creation_allowed_for_system`, so it'll fall back to the superclass, which will check this option. We generally ask to declare options in separate PRs before making code changes that use the options.

Reply:
Thanks for the heads up, I'll open up a separate PR to declare these options then

Reply:
It should be fine to register and use an option in the same pr. We used to have to deploy them separately so that ops could set a default for performance reasons, but that issue was fixed a while ago.

Reply:
@wedamija noted, thanks, I'll update the docs!

Reply:
Nice, TIL, that was a pain declaring them separately.